RunForrest/dist/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html"; charset="UTF-8" />    
+<title>Diseño Web</title>
+<link href="main.css" rel="stylesheet" type="text/css">    
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.1.0/mustache.min.js"></script>
+<script type="text/javascript" src="js/main.js"></script>
+</head>
+<body class="page" onload="callPromise()">
+    <nav class="navbar">
+        <div class="navbar__logo">
+            <img src="images/logo.png" alt="logo">
+        </div>
+        <div class="navbar-menu">
+            <div class="navbar-menu__rectangle"></div>
+            <div class="navbar-menu__rectangle"></div>
+            <div class="navbar-menu__rectangle"></div>
+            <div class="navbar-menu__text">
+                <div class="linkAgents">Directorio de Agentes</div>
+                <div class="linkContact">Contacto</div>
+                <div class="linkIngresar">Ingresar</div>
+            </div>
+        </div>
+    </nav>        
+    <div class="body-content">
+        <div class="header">
+            <div class="frame"></div>
+            <div class="main-title">
+                Los Mejores agentes de seguridad
+            </div>
+            <div class="search-bar">
+                <input type="text" class="search-bar__title" placeholder="Buscar por Ubicacion" name="inpuSearch" onkeypress="validate(event)">
```

Comment:
Same. Don't use those inline attributes to call JS methods. Use event handlers in external files instead.